pkg/compose/up.go

```diff
@@ -76,18 +77,21 @@ func (s *composeService) Up(ctx context.Context, project *types.Project, options
 		<-signalChan
 		printer.Cancel()
 		fmt.Println("Gracefully stopping... (press Ctrl+C again to force)")
+		stopC <- struct{}{}
 		stopFunc() //nolint:errcheck
+		printer.Stop()
 	}()
 
 	var exitCode int
 	eg, ctx := errgroup.WithContext(ctx)
 	eg.Go(func() error {
 		code, err := printer.Run(context.Background(), options.Start.CascadeStop, options.Start.ExitCodeFrom, stopFunc)
+		fmt.Println("printer is done")
```

Comment:
Is this log needed?

Reply:
nope, reminds from diagnostic phase trying to understand the deadlock cause :)